convert2rhel/actions/report.py

```diff
@@ -174,13 +211,14 @@ def summary(results, include_all_reports=False, with_colors=True):
             report.append(format_report_section_heading(combined_result["level"]))
             last_level = combined_result["level"]
 
-        entry = format_action_status_message(
-            combined_result["level"], message_id[0], message_id[1], combined_result["message"]
-        )
-        entry = word_wrapper.fill(entry)
-        if with_colors:
-            entry = colorize(entry, _STATUS_TO_COLOR[combined_result["level"]])
-        report.append(entry)
+        entry = format_action_status_message(combined_result["level"], message_id[0], message_id[1], combined_result)
+        paragraphs = wrap_paragraphs(entry, width=terminal_size[0])
+
+        for paragraph in paragraphs:
+            entry = word_wrapper.fill(paragraph)
+            if with_colors:
+                entry = colorize(entry, _STATUS_TO_COLOR[combined_result["level"]])
+            report.append(entry)
```

Comment:
I haven't looked at the output, but I think this needs to be changed a little:

- Each report entry should correspond to one log message rather than one paragraph.
- We can color for one entry rather than for each paragraph.
- I thnk it then makes sense to have wrap_paragraphs do the `word_wrapper.fill()` and return the wrapped string (then its API becomes single string in, single string out).  So this part would look like:

```suggestion
        entry = wrap_paragraphs(entry, width=terminal_size[0])

        if with_colors:
            entry = colorize(entry, _STATUS_TO_COLOR[combined_result["level"]])
        report.append(entry)
```

Reply:
copied this over to #888